### PR TITLE
KEYCLOAK-10776 Add info about session length in authnStatement to docs

### PR DESCRIPTION
**Pull request by mhajas**

### CODE REVIEW

**Comment by hmlnarik** [comment]
Related to keycloak/keycloak#6161

File: server_admin/topics/clients/client-saml.adoc [hunk 1]
```diff
@@ -50,8 +50,9 @@ Consent Required::
   If you've ever done a social login to Google, you'll often see a similar page.  {project_name} provides the same functionality.
 
 Include AuthnStatement::
-  SAML login responses may specify the authentication method used (password, etc.) as well as a timestamp of the login.
-  Setting this to on will include that statement in the response document. 
+  SAML login responses may specify the authentication method used (password, etc.) as well as a timestamps of the login and the session expiration.
+  This is enabled by default value, which means that AuthStatement element will be included in login responses. Notice, that setting this to off
```
Comment (hmlnarik) [inline server_admin/topics/clients/client-saml.adoc]:
```suggestion
  This is enabled by default which means that `AuthStatement` element will be included in login responses. Note that setting this to off
```

File: server_admin/topics/clients/client-saml.adoc [hunk 2]
```diff
@@ -50,8 +50,9 @@ Consent Required::
...
+  will make client side unable to know session length, what will result in timeless session on client side.
```
Comment (hmlnarik) [inline server_admin/topics/clients/client-saml.adoc]:
```suggestion
  would prevent client from determining maximum session length which could result into never expiring client session.
```

**Review by hmlnarik — CHANGES_REQUESTED** [review]

**Review by hmlnarik — DISMISSED** [review]

File: server_admin/topics/clients/client-saml.adoc [hunk 3]
```diff
@@ -50,8 +50,9 @@ Consent Required::
   If you've ever done a social login to Google, you'll often see a similar page.  {project_name} provides the same functionality.
 
 Include AuthnStatement::
-  SAML login responses may specify the authentication method used (password, etc.) as well as a timestamp of the login.
-  Setting this to on will include that statement in the response document. 
+  SAML login responses may specify the authentication method used (password, etc.) as well as a timestamps of the login and the session expiration.
+  This is enabled by default, which means that `AuthStatement` element will be included in login responses. Note that setting this to off
+  would prevent client from determining maximum session length which could result into never expiring client session.
```
Comment (andymunro) [inline server_admin/topics/clients/client-saml.adoc]: Suggest adding "the" --- would prevent "the" client from determining "the" maximum....

**Comment by mhajas** [comment]
Thank you very much for review @andymunro. I added them.

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**